### DISCUSSION TITLE
demo.livewire-powergrid.com down/inaccessible/broken

### DISCUSSION THREAD

**Discussion by bellwood** [Power-Components/livewire-powergrid#1419] [General]
Browsers wont resolve `demo.livewire-powergrid.com` but I can resolve it on the CLI:

```
[~] dig +short A demo.livewire-powergrid.com
45.77.87.102:8000.
```

I *can* load http://45.77.87.102:8000/ in my browser but quite a few of the examples throw torchlight exceptions.

**Comment by dansysanalyst** [comment]
@bellwood I could access it without problems, do you still have issues loading the page?

It also seems okay on [https://dnschecker.org](https://dnschecker.org/#A/demo.livewire-powergrid.com).
